examples/react/file-upload/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
-    "@w3ui/react-keyring": "^2.0.0",
-    "@w3ui/react-uploader": "^3.0.0",
+    "@w3ui/react-keyring": "workspace:^2.0.0",
+    "@w3ui/react-uploader": "workspace:^3.0.0",
```

Comment:
Do we need the version numbers here? We don't use in other web3.storage projects.

Reply:
oo good call, looks like we don't - I'll remove those